ATTNTestEventUtils.h

```diff
@@ -18,6 +18,7 @@
 #import "ATTNOrder.h"
 #import "ATTNPrice.h"
 #import "ATTNCart.h"
+#import "ATTNInfoEvent.h"
```

Comment:
This class needs to be moved to the `Tests` directory. Will do this in a separate PR.